snowflake/src/main/java/org/apache/iceberg/snowflake/SnowflakeCatalog.java

```diff
@@ -45,6 +45,7 @@ public class SnowflakeCatalog extends BaseMetastoreCatalog
     implements Closeable, SupportsNamespaces, Configurable<Object> {
   private static final String DEFAULT_CATALOG_NAME = "snowflake_catalog";
   private static final String DEFAULT_FILE_IO_IMPL = "org.apache.iceberg.io.ResolvingFileIO";
+  private static final String APP_IDENTIFIER = "iceberg-SDK";
```

Comment:
I don't feel like this identifier makes a lot of sense.  What is the `SDK` in this context?  The snowflake catalog isn't an SDK, so I'm not sure what this is intended to convey.

Reply:
I feel like sometime more like `iceberg-snowflake-catalog` would be more inline with what is actually connecting to snowflake.

Reply:
Agree `iceberg-snowflake-catalog` would be a better name. Switched to that instead.